content/blog/2020-06-29-optimizing-dvc-remotes.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Optimizing DVC Remotes
+date: 2020-06-29
+description: |
+  An overview of how syncing data to and from remote storage is optimized in DVC.
+picture: 2020-05-04/owl.png
```

Comment:
placeholder image

Reply:
I have the impression if you leave it blank it uses a default img BTW.

Reply:
I was having issues running the dev server (via `yarn develop`) when `picture` was unset, maybe that's just some problem with my local environment though?

Reply:
TBH I'm not sure exactly how the blog engine works! You can create a bug report though and Ivan or Roger will probably answer to that 🙂